bundles/org.openhab.binding.frenchgovtenergydata/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -1,119 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<thing:thing-descriptions bindingId="frenchgovtenergydata"
```

Comment:
Take care, this PR is even removing something from another binding!
Probably a mistake.

Reply:
Please restore this.